src/utils/stringUtils.ts

```diff
@@ -15,7 +15,51 @@ class stringUtils {
      * @param replace String to replace keywords with.
      * @returns Replaced String.
      */
-    public static async customReplace(string: string, keyword: string, replace: string) {
+     public static async customReplace(string: string, keyword: string, replace: string) {
         return string.replace(new RegExp(keyword, "g"), replace);
     }
+
+    /**
+     * ### URL Parse
+     * @param string URL that want to be parsed
+     * @returns Dictionary
+     * {
+     *  URLData: origin URL result,
+        searchParams: Search parms, (To get parms, use .get("key") method)
+        spitted: Splitted URL (['https://','','polytoria.com'])
+     * }
+    */
+
+    public static async URLParse(string: string) {
+        let url = new URL(string);
+        let searchParams_url = url.searchParams
+        let spitted_url:Array<any> = string.split("/")
+
+        return {
+            URLData: url,
+            searchParams: searchParams_url,
+            spitted: spitted_url
+        }
+    }
+
+    /**
+     * ### Get Numbers From URL
+     * @param string URL that want to be parsed
+     * @returns array Numbers that has been found in URL
+    */
+    public static async GetNumbersFromURL(string: string) {
+        let result = await stringUtils.URLParse(string)
+        let splitted = result.spitted
+
+        let returnresult:Array<any> = []
+
+        splitted.forEach(function (item:string) {

```

Comment:
Use ES6 Syntax, this is typescript not JavaScript.

```js
splitted.forEach(item => {})
```

`number` type is inherited from the array, meaning it should always be a number.

Reply:
Thus no need for types on `item` which is not a string?